images/metering-ansible-operator/roles/meteringconfig/tasks/configure_networking.yml

```diff
@@ -68,4 +68,4 @@
     _ocp_enabled_global_proxy_http_proxy: "{{ _cluster_proxy_config.status.httpProxy }}"
     _ocp_enabled_global_proxy_https_proxy: "{{ _cluster_proxy_config.status.httpsProxy }}"
   no_log: true
-  when: cluster_proxy_config is defined and cluster_proxy_config.resources | length > 0 and cluster_proxy_config.status is defined
+  when: cluster_proxy_config is defined and (cluster_proxy_config.resources | length) and _cluster_proxy_config.status is defined
```

Comment:
Something is off about this conditional (from the metering-operator Pod logs):
```
TASK [meteringconfig : Check if the cluster-wide Proxy configured] *************
[1;30mtask path: /opt/ansible/roles/meteringconfig/tasks/configure_networking.yml:33[0m
Wednesday 08 April 2020  21:17:58 +0000 (0:00:00.048)       0:00:07.940 ******* 
[0;32mok: [localhost] => {"changed": false, "resources": [{"apiVersion": "config.openshift.io/v1", "kind": "Proxy", "metadata": {"creationTimestamp": "2020-04-08T20:38:32Z", "generation": 1, "name": "cluster", "resourceVersion": "456", "selfLink": "/apis/config.openshift.io/v1/proxies/cluster", "uid": "09160338-f4e6-4f1f-81bb-ecf077de1713"}, "spec": {"trustedCA": {"name": ""}}, "status": {}}]}[0m

TASK [meteringconfig : set_fact] ***********************************************
[1;30mtask path: /opt/ansible/roles/meteringconfig/tasks/configure_networking.yml:48[0m
Wednesday 08 April 2020  21:17:58 +0000 (0:00:00.866)       0:00:08.806 ******* 
[0;31mfatal: [localhost]: FAILED! => {"censored": "the output has been hidden due to the fact that 'no_log: true' was specified for this result"}[0m
```

Reply:
```suggestion
  when:
  - cluster_proxy_config is defined
  - cluster_proxy_config.resources | length > 0
  - _cluster_proxy_config.status is defined
  - _cluster_proxy_config.status | length > 0

```

Reply:
hey just curious... why is one cluster_proxy_config and one _cluster_proxy_config? Thanks